<!DOCTYPE html>
<html>
	<head>
		<meta charset="utf-8"/>
		<link rel="icon" href="data:;base64,iVBORw0KGgo=">
		<title>Gene Graphics</title>
		<link rel="stylesheet" type="text/css" href="styles/tutorials.css">
		<link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
	</head>
	<body>
		<h1>Tutorial 2: Genbank File</h1>
		<div class="bgcolored">
		 <h3>Table of Contents</h3>
            <p>
                 <ul>
                     <li>
                         <a href="#fetching">Fetching Data</a>
                     </li>
                     <li>
                         <a href="#modifying">Modifying</a>
                     </li>
                 </ul>
				 <p>Final files that can be obtained by following this tutorial: <a href="data/tutorial2/tutorial2_finalproduct.svg" target="_blank">SVG</a> <a href="data/tutorial2/tutorial2_finalproduct.png" target="_blank">PNG</a> <a href="data/tutorial2/tutorial2_finalproduct.tsv" target="_blank">TSV</a>
           </p>

		</div>
		<div class="bgcolored"><p>In this tutorial, we will produce a simple figure showing a similar region in two genomes by accessing data from NCBI Genbank. The figure we will make is inspired by a figure in the publication Thiaville Jennifer J., et al., 2016 as <b>Figure 2A</b>. You can access it here: <a href="http://journal.frontiersin.org/article/10.3389/fmicb.2016.00431/full" target="_blank">http://journal.frontiersin.org/article/10.3389/fmicb.2016.00431/full</a> </p>
		<p>Thiaville Jennifer J., Frelin Océane, García-Salinas Carolina, Harrison Katherine, Hasnain Ghulam, Horenstein Nicole A., Díaz de la Garza Rocio I., Henry Christopher S., Hanson Andrew D., de Crécy-Lagard Valérie, 2016. Experimental and Metabolic Modeling Evidence for a Folate-Cleaving Side-Activity of Ketopantoate Hydroxymethyltransferase (PanB). Frontiers in Microbiology. 7, 431. doi:10.3389/fmicb.2016.00431</p>
		<p>The image below is the image from the paper.</p></div>
		<img src="images/tutorial2/imgpublished.png">
		<div class="bgcolored"><p>For simplicity, we will only be comparing the first two regions found in the image. The final image that we will be produce is shown below.</p></div>
		<img src="images/tutorial2/imgfinal.png">
		<div class="bgcolored"><h2><a name="fetching">Fetching Data</a></h2><p>We will show a comparison between the neighborhoods of the <em>folK</em> gene in two genomes, <em>Acinetobacter baylyi</em> sp. ADP1 and <em>Salmonella enterica</em> strain LT2. Navigate to the "Gene Graphics" app at <a href="http://www.katlabs.cc/genegraphics/" target="_blank">http://katlabs.cc/genegraphics/</a>. Then, click on "Input" and "Fetch from NCBI". This option allows you to quickly import a whole genome or a specific region of a genome from NCBI Genbank.</p></div>
		<img src="images/tutorial2/img0.png">
		<div class="bgcolored"><p>Search by Genome location.  The Import tool can take genome name or Genbank ID.  Enter "Acinetobacter sp." and notice how it narrows down the search as you type.  You could simply type "ADP1" and get the result directly, as well.  You can click on the genome that you want at any time.  Use region 2986058 to 2991851.</p></div>
		<img src="images/tutorial2/img1.png">
		<div class="bgcolored"><p>After the data is imported, you should see the following region on your screen:</p></div>
		<img src="images/tutorial2/img2.png">
		<div class="bgcolored"><p>Sometimes, you might want the reverse complement of region accessed from Genbank. The <em>Salmonella enterica</em> strain LT2 genome found in NCBI Genbank by default displays the region we are interested in reversed from the <em>Acinetobacter baylyi</em> region. We can use the NCBI interface to export a file that will show us the region in reverse complement, then upload it to the Gene Graphics tool. In a new tab or window, access the NCBI database homepage at <a href="https://www.ncbi.nlm.nih.gov/nucleotide/" target="_blank">https://www.ncbi.nlm.nih.gov/nucleotide/</a>. Type "Salmonella enterica strain LT2" into the search box. Click "Search".</p></div>
		<img src="images/tutorial2/img3.png">
		<div class="bgcolored"><p>Click on the result "Salmonella enterica strain LT2, complete genome". It will bring you to the page <a href="https://www.ncbi.nlm.nih.gov/nuccore/CP014051.1" target="_blank"> https://www.ncbi.nlm.nih.gov/nuccore/CP014051.1</a>.</p></div>
		<img src="images/tutorial2/img4.png">
		<div class="bgcolored"><p>You will see a standard Genbank file. On the right, you will see a box that says "Change region shown". Click to open the options and choose "Selected region". The region we want for this example is from 583556 to 590090. Click "Update View". Then, under "Customize View", select the option "Show reverse complement" and click the "Update View" button underneath.</p></div>
		<img src="images/tutorial2/img5.png">
		<div class="bgcolored"><p>Now we can export our customized Genbank file. Select "Send" and choose "Complete Record", "File", the Genbank format, and click "Create File" to download it. The file is also available for download <a href="data/tutorial2/Salmonella_enterica_strLT2.gb" target="_blank">here</a>.</p></div>
		<img src="images/tutorial2/img6.png">
		<div class="bgcolored"><p>Next, return to the Gene Graphics tab and click "Import a File". Follow the dialog options to upload your Genbank file.</p></div>
		<img src="images/tutorial2/img7.png">
		<div class="bgcolored"><h2><a name="modifying">Modifying</a></h2><p>Now that both regions are uploaded into the tool, the image can be edited. You may have noticed that our first region has an extra partial gene at the beginning. This is because this gene overlaps the first part of our region by a few bases. We can simply delete the gene by clicking on it and then selecting the trash can icon in the resulting window.</p></div>
		<img src="images/tutorial2/img8.png">
		<div class="bgcolored"><p>Decrease the width of the image by clicking on "Graph Settings", then "Size settings", and entering 1000 for the "Image Width". Change the height of the genes by decreasing "Gene height" to 30.  The resulting image will update when you click Update or tab away from the fields.</p></div>
		<img src="images/tutorial2/img9.png">
		<div class="bgcolored"><p>Under "Graph Settings", then "Lane settings", toggle the option "Overlapping genes move to a new lane", and toggle the option so that genes may not overlap each other.</p></div>
		<img src="images/tutorial2/img10.png">
		<div class="bgcolored"><p>Under "Graph Settings", then "Gene shape", toggle the option to change the symbols to tags instead of arrows.</p></div>
		<img src="images/tutorial2/img10a.png">
		<div class="bgcolored"><p>Some of the genes in the region we selected have an unknown or irrelevant function, so we should use a neutral color such as gray. Click on the first gene in the <em>Acinetobacter</em> sp. ADP1 region and select a gray color. Edit the text to remove the label. Then, click on the "Copy Gene Colors" button. This will copy the Gene and Label colors of this gene so that you can paste it to the other genes with unknown or irrelevant functions.</p></div>
		<img src="images/tutorial2/img11.png">
		<div class="bgcolored"><p>Click or right click on the last and second to last genes in the <em>Acinetobacter</em> sp. ADP1 region, and the first and last gene in the <em>Salmonella enterica</em> strain LT2 region, then select Paste to apply the color to all of them. Note that genes will be outlined with red while they are selected as a destination for the paste.</p></div>
		<img src="images/tutorial2/img13.png">
		<div class="bgcolored"><p>The folK gene in <em>Acinetobacter</em> sp. ADP1 was not annotated as folK, and its "product" is annotated slightly differently than the folK gene in <em>Salmonella enterica</em> strain LT2. This results in the two genes having different colors, but we would like the colors to be the same. Select folK in <em>Acinetobacter</em> sp. ADP1 and Copy to the second folK as shown below.</p></div>
		<img src="images/tutorial2/img15.png">
		<div class="bgcolored"><p>Continue naming and editing the colors of the genes as shown in the following image.</p></div>
		<img src="images/tutorial2/img19.png">
		<div class="bgcolored"><p>Next, change the label positions so that they are above the genes by selecting "Label Settings", "Gene Labels" and then selecting the "Above" option.</p></div>
		<img src="images/tutorial2/img20.png">
		<div class="bgcolored"><p>Change the font style to italic by choosing "Italic".</p></div>
		<img src="images/tutorial2/img21.png">
		<div class="bgcolored"><p>The final produced image can be exported as a SVG file or as a PNG file, by clicking on the “Export” button. The edits done in the “Gene Graphics” application can also be exported as a TSV file, which will produce a table. This can be useful if an image is started, but is not yet finished. Reloading the produced TSV file will allow you to continue your edits.</p> 
		<img src="images/tutorial2/img23.png">

	</body>
</html>
