<!DOCTYPE html>
<html>
	<head>
		<meta charset="utf-8"/>
		<link rel="icon" href="data:;base64,iVBORw0KGgo=">
		<title>Gene Graphics</title>
		<link rel="stylesheet" type="text/css" href="styles/tutorials.css">
		<link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
	</head>
	<body>
		<h1>Tutorial 1: DUF89 Subfamily I Neighborhoods</h1>
		<div class="bgcolored"><p>In this tutorial, we are going to reproduce a figure similar to one that has been published. The figure to recreate can be found in the supplemental material associated with the publication Huang, et al., 2016, as <b>Supplementary Figure 4B</b>. You can access it here: <a href=http://www.nature.com/nchembio/journal/v12/n8/abs/nchembio.2108.html#supplementary-information" target="_blank">http://www.nature.com/nchembio/journal/v12/n8/abs/nchembio.2108.html#supplementary-information</a></p>
		<p>Huang, L., Khusnutdinova, A., Nocek, B., Brown, G., Xu, X., Cui, H., Petit, P., Flick, R., Zallot, R., Balmant, K., Ziemak, M.J., Shanklin, J., de Crécy-Lagard, V., Fiehn, O., Gregory, J.F., Joachimiak, A., Savchenko, A., Yakunin, A.F., Hanson, A.D., 2016. A family of metal-dependent phosphatases implicated in metabolite damage-control. Nat. Chem. Biol. 12, 621–7. doi:10.1038/nchembio.2108</p>
		<p>The image presented below is the one we will reproduce.</p></div>
		<img src="images/tutorial1/img0.png">
		<div class="bgcolored"><p>The image presented below is the final product generated with Gene Graphics, following this tutorial.</p></div>
		<img src="images/tutorial1/imgfinal.png">
		<div class="bgcolored"><p>The original publication identified a new family of phosphatases. With this Figure, we want to illustrate that genes encoding the phosphatase DUF89 are often found closely associated, on the genome of several non-closely related bacteria, with genes involved in the purine synthesis (i.e. purM, purF, GuaB, purE, IMPC and AICARF).</p>
		<p>For this tutorial, we are going use the public SEED database (pubSEED) to obtain the necessary genome information to create the Figure. Access the SEED database at <a href="http://pubseed.theseed.org" target="_blank">http://pubseed.theseed.org</a>.</p></div>
		<img src="images/tutorial1/img1.png">
		<div class="bgcolored"><p>From the homepage, use the search box to obtain the needed information about the gene DUF89 from Methanococcus vannieli. The first result for this search is the gene we are looking for, that encodes the protein “DUF89 protein CxxC subfamily”, from the genome Methanococcus vannieli vannielii SB.</p></div>
		<img src="images/tutorial1/img2.png">
		<div class="bgcolored"><p>Browsing to the associated FIG page (<a href="http://pubseed.theseed.org/?page=Annotation&feature=fig%7C406327.7.peg.591" target="_blank">fig|406327.7.peg.591</a>), will bring you to the individual gene page information. Going down, on the webpage, once the “Compare Regions” has loaded, the neighboring genes of our gene of interest can be visualized. The red arrow represents the queried gene, while colored arrows represent conserved neighboring genes and grey arrows correspond to genes that are not conserved among the genomes displayed.</p></div>
		<img src="images/tutorial1/img3.png">
		<div class="bgcolored"><p>To be able to extract the information needed, we need to access the advanced tab, and edit a few settings.</p></div>
		<img src="images/tutorial1/img4.png">
		<div class="bgcolored"><p>We need to have shown to us, more genomes on the “Compare Regions” area, by changing the number of regions (similar to genome) to 1000. In addition, we want to focus on the defined orthologous families “PGfams”, by switching the “Pinned CDS selection” to the appropriate setting, and then press the “update with selected” button.</p>
		<p>We are now going to need to selected the genomes of interest, for our figure. We basically want to have genomes from taxonomically diverse bacteria, to show that the gene relations are well conserved. Because by default, all genomes are selected, we need to unselect these by clicking the “uncheck all” button.</p>
		<p>We are then going to select the following genomes:<br>
				M. vannieli van (Methanococcus vannieli vannielii SB)<br>
				M. maripaludis (Methanococcus maripaludis C6)<br>
				M. jannaschii D (Methanocaldococcus jannaschii DSM 2661)<br>
				M. stadtmanae D (Methanosphaera stadtmanae DSM 3091)<br>
				T. ammonificans (Thermovibrio ammonificans HB-1)<br>
				C. mediatlantic (Caminibacter mediatlanticus TB-2)<br>
				N. sp. SB155-2 (Nitratiruptor sp. SB155-2)<br>
				L. bacterium 1_ (Lachnospiraceae bacterium 1 4 56FAA)<br>
				H. cinaedi ATCC (Helicobacter cinaedi ATCC BAA-847)<br>
				And press the “update with selected” button.</p>
		<p>The following selection is obtained:</p></div>
		<img src="images/tutorial1/img6.png">
		<div class="bgcolored"><p>To focus on the close neighbors of our gene of interest (red arrow), zooming in, in the genome browser, is useful. The “Region size” is reduced to 6000 bp.</p></div>
		<img src="images/tutorial1/img7.png">
		<div class="bgcolored"><p>The information regarding the positions of all the genes presented in the image is available, in a tabular format, in the “Tabular Region Information” tab.</p></div>
		<img src="images/tutorial1/img8.png">
		<div class="bgcolored"><p>We need to export the table as a TSV file (Tab-separated values). The exported table can be directly obtained <a href="data/tutorial5/tutorial1_SEEDexport.tsv" target="_blank">here</a>.</p>
		<p>The exported file will now be uploaded to “Gene Graphics” at <a href="http://katlabs.cc/genegraphics">http://katlabs.cc/genegraphics/</a>.</p>
		<p>In “Gene Graphics”, load the edited table by going to the “Input” tab, and then press the “Choose a file” button, and then, upload the exported file.</p></div>
		
	</body>
</html>
