<!DOCTYPE html>
<html>
	<head>
		<meta charset="utf-8"/>
		<link rel="icon" href="data:;base64,iVBORw0KGgo=">
		<title>Gene Graphics</title>
		<link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='styles/tutorials.css')}}">
		<link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
	</head>
	<body>
		<h1>Tutorial 3: DUF89 Subfamily I Neighborhoods</h1>
		<div class="bgcolored">
		 <h3>Table of Contents</h3>
				<p>
					 <ul>
						 <li>
							<a href="#fetching">Fetching Data</a>
						 </li>
						 <li>
							 <a href="#modifying">Modifying</a>
						 </li>
					 </ul>
					 Final files that can be obtained by following this tutorial: <a href="{{ url_for('static', filename='data/tutorial3/tutorial3_finalproduct.svg') }}" target="_blank">SVG</a> <a href="{{ url_for('static', filename='data/tutorial3/tutorial3_finalproduct.png') }}" target="_blank">PNG</a> <a href="{{ url_for('static', filename='data/tutorial3/tutorial3_finalproduct.tsv') }}" target="_blank">TSV</a></p>
			</div>
		<div class="bgcolored"><p>In this tutorial, we are going to reproduce a figure similar to one that has been published. The figure to recreate can be found in the supplemental material associated with the publication Huang, et al., 2016, as <b>Supplementary Figure 4B</b>. You can access it here: <a href=http://www.nature.com/nchembio/journal/v12/n8/abs/nchembio.2108.html#supplementary-information" target="_blank">http://www.nature.com/nchembio/journal/v12/n8/abs/nchembio.2108.html#supplementary-information</a></p>
		<p>Huang, L., Khusnutdinova, A., Nocek, B., Brown, G., Xu, X., Cui, H., Petit, P., Flick, R., Zallot, R., Balmant, K., Ziemak, M.J., Shanklin, J., de Crécy-Lagard, V., Fiehn, O., Gregory, J.F., Joachimiak, A., Savchenko, A., Yakunin, A.F., Hanson, A.D., 2016. A family of metal-dependent phosphatases implicated in metabolite damage-control. Nat. Chem. Biol. 12, 621–7. doi:10.1038/nchembio.2108</p>
		<p>The image presented below is the one we will reproduce.</p></div>
		<img src="{{ url_for('static', filename='images/tutorial3/img0.png') }}">
		<div class="bgcolored"><h2><a name="fetching">Fetching Data</a></h2><p>The original publication identified a new family of phosphatases. With this Figure, we want to illustrate that genes encoding the phosphatase DUF89 are often found closely associated, on the genome of several non-closely related bacteria, with genes involved in the purine synthesis (i.e. purM, purF, GuaB, purE, IMPC and AICARF).</p>
		<p>For this tutorial, we are going use the public SEED database (pubSEED) to obtain the necessary genome information to create the Figure. Access the SEED database at <a href="http://pubseed.theseed.org" target="_blank">http://pubseed.theseed.org</a>.</p></div>
		<img src="{{ url_for('static', filename='images/tutorial3/img1.png') }}">
		<div class="bgcolored"><p>From the homepage, use the search box to obtain the needed information about the gene DUF89 from <em>Methanococcus vannieli</em>. The first result for this search is the gene we are looking for, that encodes the protein “DUF89 protein CxxC subfamily”, from the genome <em>Methanococcus vannieli</em> vannielii SB.</p></div>
		<img src="{{ url_for('static', filename='images/tutorial3/img2.png') }}">
		<div class="bgcolored"><p>Browsing to the associated FIG page (<a href="http://pubseed.theseed.org/?page=Annotation&feature=fig%7C406327.7.peg.591" target="_blank">fig|406327.7.peg.591</a>), will bring you to the individual gene page information. Going down, on the webpage, once the “Compare Regions” has loaded, the neighboring genes of our gene of interest can be visualized. The red arrow represents the queried gene, while colored arrows represent conserved neighboring genes and grey arrows correspond to genes that are not conserved among the genomes displayed.</p></div>
		<img src="{{ url_for('static', filename='images/tutorial3/img3.png') }}">
		<div class="bgcolored"><p>To be able to extract the information needed, we need to access the advanced tab, and edit a few settings.</p></div>
		<img src="{{ url_for('static', filename='images/tutorial3/img4.png') }}">
		<div class="bgcolored"><p>We need to have shown to us, more genomes on the “Compare Regions” area, by changing the number of regions (similar to genome) to 1000. In addition, we want to focus on the defined orthologous families “PGfams”, by switching the “Pinned CDS selection” to the appropriate setting, and then press the “update with selected” button.</p>
		<p>We are now going to need to selected the genomes of interest, for our figure. We basically want to have genomes from taxonomically diverse bacteria, to show that the gene relations are well conserved. Because by default, all genomes are selected, we need to unselect these by clicking the “uncheck all” button.</p>
		<p>We are then going to select the following genomes:<br>
			<em>M. vannieli van</em> (<em>Methanococcus vannieli</em> vannielii SB)<br>
			<em>M. maripaludis</em> (<em>Methanococcus maripaludis</em> C6)<br>
			<em>M. jannaschii</em> D (<em>Methanocaldococcus jannaschii</em> DSM 2661)<br>
			<em>M. stadtmanae</em> D (<em>Methanosphaera stadtmanae</em> DSM 3091)<br>
			<em>T. ammonificans</em> (<em>Thermovibrio ammonificans</em> HB-1)<br>
			<em>C. mediatlantic</em> (<em>Caminibacter mediatlanticus</em> TB-2)<br>
			<em>N.</em> sp. SB155-2 (<em>Nitratiruptor</em> sp. SB155-2)<br>
			<em>L. bacterium</em> 1_ (<em>Lachnospiraceae bacterium</em> 1 4 56FAA)<br>
			<em>H. cinaedi</em> ATCC (<em>Helicobacter cinaedi</em> ATCC BAA-847)<br>
				And press the “update with selected” button.</p>
		<p>The following selection is obtained:</p></div>
		<img src="{{ url_for('static', filename='images/tutorial3/img6.png') }}">
		<div class="bgcolored"><p>To focus on the close neighbors of our gene of interest (red arrow), zooming in, in the genome browser, is useful. The “Region size” is reduced to 6000 bp.</p></div>
		<img src="{{ url_for('static', filename='images/tutorial3/img7.png') }}">
		<div class="bgcolored"><p>The information regarding the positions of all the genes presented in the image is available, in a tabular format, in the “Tabular Region Information” tab.</p></div>
		<img src="{{ url_for('static', filename='images/tutorial3/img8.png') }}">
		<div class="bgcolored"><p>We need to export the table as a TSV file (Tab-separated values) that we will load into Gene Graphics. The exported table can be directly obtained <a href="{{ url_for('static', filename='data/tutorial3/tutorial3_SEEDexport.tsv') }}" target="_blank">here</a>.</p>
		<p>In Gene Graphics, load the edited table by going to the “Input” tab, and then press the “Import a file” button, and then, upload the table file.</p></div>
		<img src="{{ url_for('static', filename='images/tutorial3/img09.png') }}">
		<div class="bgcolored"><h2><a name="modifying">Modifying</a></h2><p>Each genome has more genes in it then we need. To start, we should identify the DUF89 gene.  As you pass the mouse over each gene, the readout in the upper right will give details about it.  Passing over "Gene_5" will show "DUF89 protein CxxC subfamily".</p></div>
		<img src="{{ url_for('static', filename='images/tutorial3/img10.png') }}">
		<div class="bgcolored"><p>Click on the gene to edit the text and change it to DUF89.  At this time we can also change the gene's color to match the published image.</p></div>
		<img src="{{ url_for('static', filename='images/tutorial3/img11.png') }}">
		<div class="bgcolored"><p>Comparing "Methanococcus vannieli vannielii SB" to the original image, we only need DUF89 and one gene to the right.  Selecting a gene and then click on the trash can image will allow you to delete it, after a prompt.  Delete the genes we do not need from this genome.</p></div>
		<img src="{{ url_for('static', filename='images/tutorial3/img12.png') }}">
		<div class="bgcolored"><p>Each of the remaining genomes has a DUF89 gene.  You should note that the default color for the gene is based on its descriptive label, so all of the DUF89 genes are purple and easy to identify.  We can use the copy feature to apply the text and color settings of our edited DUF89 gene to the remainder of the genomes.  Select the edited DUF89 gene, then click the copy button.  You will select each of the genes you wish to apply the settings and text to.  They will be outlined in red.  When finished selecting all destinations, click paste.</p></div>
		<img src="{{ url_for('static', filename='images/tutorial3/img13.png') }}">
		<div class="bgcolored"><p>Next, use the published image to determine which genes from each genome we require, and select and delete those that are not needed.  For example, for "Methanococcus maripaludis C6" we need only the genes to either side of DUF89.  Delete the genes that are not needed for each genome.</p></div>
		<img src="{{ url_for('static', filename='images/tutorial3/img14.png') }}">
		<div class="bgcolored"><p>Now we can edit the remaining genes to match the published image.  Some of the other genes are part of more than one genome - we can use the copy tool to easily label them as well.</p></div>
		<img src="{{ url_for('static', filename='images/tutorial3/img15.png') }}">
		<img src="{{ url_for('static', filename='images/tutorial3/img16.png') }}">
		<div class="bgcolored"><p>There are a number of genes that need to be blank.  Simply edit one to have no label and a white background and paste it onto the other genes that need this display. </p></div>
		<img src="{{ url_for('static', filename='images/tutorial3/img17.png') }}">
		<div class="bgcolored"><p>Use Graph Settings -> Size Settings to set the size of the image.  A width of 500px will bring the genes to a similar size as the published image. </p></div>
		<img src="{{ url_for('static', filename='images/tutorial3/img18.png') }}">
		<div class="bgcolored"><p>Finally, we should edit the text of the DUF89 genes to match the published image.  The text editor can set the font to bold, italics, and the text color to white.</p></div>
		<img src="{{ url_for('static', filename='images/tutorial3/img19.png') }}">
		<div class="bgcolored"><p>The final produced image can be exported, as a SVG file or as a PNG file, by clicking on the “Export” button. The edits done in the “Gene Graphics” application, can also be exported as a TSV file. This can be useful if an image is started, but is not yet finished. Reloading the produced TSV file will allow you to continue your edits.</p>
		</div>
	</body>
</html>
